monkey/agent_plugins/exploiters/snmp/src/plugin.py

```diff
@@ -120,6 +120,7 @@ def _create_snmp_exploiter(
         run_options = LinuxRunOptions(
             agent_destination_path=destination_path,
             dropper_execution_mode=DropperExecutionMode.NONE,
+            include_otp=False,
```

Comment:
If we don't include the otp in the script then we should have a way to set it when running the script right ? Or I am blind and missed that.

Reply:
It's included by default (the default for `include_otp` is True)